benchmark/src/jmh/java/com/knuddels/jtokkit/AbstractBenchmark.java

```diff
@@ -1,38 +1,38 @@
 package com.knuddels.jtokkit;
 
 import com.knuddels.jtokkit.api.Encoding;
-import java.util.List;
 import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.infra.Blackhole;
+
+import java.util.List;
 
 public abstract class AbstractBenchmark {
 
-	@Benchmark
-	public void benchmarkP50kBase(final BenchmarkingState state, final Blackhole blackhole) {
-		blackhole.consume(encodeAll(state.p50kBase, state.fileContents));
-	}
-
-	@Benchmark
-	public void benchmarkR50kBase(final BenchmarkingState state, final Blackhole blackhole) {
-		blackhole.consume(encodeAll(state.r50kBase, state.fileContents));
-	}
-
-	@Benchmark
-	public void benchmarkP50kEdit(final BenchmarkingState state, final Blackhole blackhole) {
-		blackhole.consume(encodeAll(state.p50kEdit, state.fileContents));
-	}
-
-	@Benchmark
-	public void benchmarkCl100kBase(final BenchmarkingState state, final Blackhole blackhole) {
-		blackhole.consume(encodeAll(state.cl100kBase, state.fileContents));
-	}
-
-	/**
-	 * Encodes all file contents with the given encoding.
-	 *
-	 * @param encoding the encoding to use
-	 * @param fileContents the file contents to encode
-	 * @return a list of encoded token lists
-	 */
-	protected abstract List<List<Integer>> encodeAll(final Encoding encoding, final List<String> fileContents);
+    @Benchmark
```

Comment:
the project contained tabs and spaces randomly, formatted the affected files to IDEA defaults

Reply:
You can ignore whitespace diffs with:
<img width="270" alt="image" src="https://github.com/knuddelsgmbh/jtokkit/assets/1841944/4daed0dc-725d-4bd8-838e-d80238167ce8">